README.md

```diff
@@ -0,0 +1,8 @@
+# FlyINN
+
+Intermodal mobility app for a sharing economy.
+
+## Product vision
```

Comment:
Maybe merge this with the introduction to FlyInn

Reply:
Handled in commit f1f8a54